### DISCUSSION TITLE
gpt-4 models?

### DISCUSSION THREAD

**Discussion by xric25** [MacPaw/OpenAI#86] [Q&A] [answered]
I haven't read anything about it and I'm wondering if I can use gpt-4 models once I've been approved by OpenAI for limited beta testing?

Thanks!

**Comment by StefanBristol** [comment, marked as answer]
Yes you can. I currently do.  Use 'query = ChatQuery(model: .gpt4, messages: ...)

**Reply by xric25** [reply]
Thank you :) .. In the meantime I also got gpt4 access and can also confirm that it works.
